parent/src/pom-template.xml

```diff
@@ -0,0 +1,285 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 https://maven.apache.org/xsd/maven-4.0.0.xsd"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
+    <modelVersion>4.0.0</modelVersion>
+    <groupId>io.micronaut</groupId>
+    <artifactId>micronaut-parent</artifactId>
+    <version>%%version%%</version>
+    <packaging>pom</packaging>
+    <parent>
+        <groupId>io.micronaut.bom</groupId>
+        <artifactId>micronaut-bom</artifactId>
+        <version>%%version%%</version>
+    </parent>
+    <name>Micronaut</name>
+    <description>Natively Cloud Native</description>
+    <url>https://micronaut.io</url>
+    <licenses>
+        <license>
+            <name>The Apache Software License, Version 2.0</name>
+            <url>http://www.apache.org/licenses/LICENSE-2.0.txt</url>
+            <distribution>repo</distribution>
+        </license>
+    </licenses>
+    <scm>
+        <url>scm:git@github.com:micronaut-projects/micronaut-core.git</url>
+        <connection>scm:git@github.com:micronaut-projects/micronaut-core.git</connection>
+        <developerConnection>scm:git@github.com:micronaut-projects/micronaut-core.git</developerConnection>
+    </scm>
+    <developers>
+        <developer>
+            <id>graemerocher</id>
+            <name>Graeme Rocher</name>
+        </developer>
+    </developers>
+    <properties>
+        <jdk.version>1.8</jdk.version>
+        <release.version>8</release.version>
+        <maven.compiler.source>${jdk.version}</maven.compiler.source>
+        <maven.compiler.target>${jdk.version}</maven.compiler.target>
+        <maven.compiler.parameters>true</maven.compiler.parameters>
+        <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+        <project.reporting.outputEncoding>UTF-8</project.reporting.outputEncoding>
+        <exec.executable>java</exec.executable>
+        <micronaut.version>%%version%%</micronaut.version>
+    </properties>
+    <profiles>
+        <profile>
+            <id>jdk-9-or-later</id>
+            <activation>
+                <jdk>[9,)</jdk>
+            </activation>
+            <properties>
+                <maven.compiler.release>${release.version}</maven.compiler.release>
+            </properties>
+        </profile>
```

Comment:
This profile shouldn't be needed anymore, how is it used?

Reply:
I have pushed a change, but I think it probably doesn't make sense to have both `<maven.compiler.release>${release.version}</maven.compiler.release>` and the source/target compatibility set. If the previous profile was using `release`, then we should probably only use `release` too.